script.js

```diff
@@ -1,5 +1,145 @@
-const test = document.getElementById('test');
+//  Mobile Menu
 
-test.addEventListener('click', () => {
-  test.style.color = 'blue';
-});
\ No newline at end of file
+const menuButton = document.querySelector('#menuButton');
+const closeMenuButton = document.querySelector('#closeMenuBtn');
+
+menuButton.addEventListener('click', () => {
+  document.getElementById('menu').style.display = 'block';
+  document.getElementById('blurSection').style.filter = 'blur(5px)';
+});
+
+closeMenuButton.addEventListener('click', () => {
+  document.getElementById('menu').style.display = 'none';
+  document.getElementById('blurSection').style.filter = 'none';
+});
+
```

Comment:
- [ ] The mobile `navbar` does not disappear upon screen resize and if is displaying will also appear in the desktop version. Please make sure that your mobile menu disappears as the screen resizes to desktop resolution.

<img width="1131" alt="Screenshot 2023-03-17 at 9 47 21 PM" src="https://user-images.githubusercontent.com/44798044/225990997-1e754bf9-dc3b-4160-ba38-9daa6549f3b4.png">